src/storage/documentService/documentService.js

```diff
@@ -68,9 +84,28 @@ const uploadDocument = async (
   };
 };
 
+const getQueueNumber = async () => {
+  const created = Math.floor(Date.now() / 1000);
+  const params = {
+    TableName: config.dynamodb.storageTableName,
+    Item: {
```

Comment:
i think its better to explicitly indicate that it's a placeholder entry in the body instead of leaving it blank and up to interpretation

Reply:
couldn`t get it? add a comment or  create another table which stores temp data and move that data to main table when everything confirms.

Reply:
`Item: {
 id: uuid(),
 created,
 ttl: created + DEFAULT_TTL,
 awaitingUpload: true
}`

for example - but will need to remember to clean up the awaitingUpload when updating later